### PR TITLE
Bug 1390206 profile

### PR DESCRIPTION
**Pull request by bclary**
The first commit changes from using a string of user_pref values in profile_ to keeping the preferences in a dictionary that is passed to FirefoxProfile to create the profile. See the bug for more details.

The second commit adds the preference prefs["layout.css.servo.enabled"] = False to prevent servo from running.

### CODE REVIEW

File: slave/executors.py [hunk 1]
```diff
@@ -132,11 +134,11 @@ def execute(self, benchmark, env, args, profile):
         # delete profile
         runner.rm("profile/")
 
-        # create new profile
-        runner.mkdir("profile/")
+        # Force stylo off.
+        prefs["layout.css.servo.enabled"] = False
```
Comment (stuartphilp) [inline slave/executors.py]: one thing i'd maybe suggest is if we can pull this out into a separate file so it's just a case of listing out prefs and desired settings (presuming we have many more experiments in the future that we might want some control over)... there's also maybe further discussion about using a service to set our automation to the correct state (maybe even allowing remote flipping so we can effectively a/b our results)

File: slave/executors.py [hunk 2]
```diff
@@ -134,6 +134,9 @@ def execute(self, benchmark, env, args, prefs):
         # delete profile
         runner.rm("profile/")
 
+        # Force stylo off.
+        prefs["layout.css.servo.enabled"] = False
```
Comment (bnjbvr) [inline slave/executors.py]: Can you put it next to the other Firefox default prefs, for consistency, please? (in `configs.py`)

Agreed using an external file or external service would be even better.

**Comment by bclary** [comment]
I made the changes and amended the commit and did a force push. I should have added the changes as a new commit and done a squash during the merge. I'll just do the merge now anyway.

**Comment by bnjbvr** [comment]
Force-pushing works great too, thanks for merging!

**Pull request merged**